Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1939,6 +1934,7 @@ extension BrowserViewController: TabManagerDelegate {
             // we therefore have to hide all the scrollViews that we are no actually interesting in interacting with
             // to ensure that scrollsToTop actually works
             wv.scrollView.hidden = true
```

Comment:
We could also then remove lines 1947 - 1949 below where we unhide the webview if it is hidden.